doc/dev_start_guide.txt

```diff
@@ -219,27 +219,12 @@ GitHub, to let them know that you have submitted a fix.
 Tips for Quality Contributions
 ==============================
 
-Coding Style Auto Check
+Coding Style
 -----------------------
 
-In Theano, we use the same coding style as the `Pylearn
-<http://deeplearning.net/software/pylearn/v2_planning/API_coding_style.html>`_
-project, except that we don't use the numpy docstring standard.
-The principal thing to know is that we follow the
-`PEP 8 <http://www.python.org/dev/peps/pep-0008/>`_ coding style.
-
-We use git hooks provided in the project `pygithooks
-<https://github.com/lumberlabs/pygithooks>`_ to validate that commits
-respect pep8. This happens when each user commits, not when we
-push/merge to the Theano repository. Github doesn't allow us to have
-code executed when we push to the repository. So we ask all
-contributors to use those hooks.
-
-For historic reason, we currently don't have all files respecting pep8.
-We decided to fix everything incrementally. So not all files respect it
-now. So we strongly suggest that you use the "increment" pygithooks
-config option to have a good workflow. See the pygithooks main page
-for how to set it up for Theano and how to enable this option.
+Please always respect the 'coding style specifications
```

Comment:
You should use backticks ` instead of '.

Please always compile the documentation locally before committing. To do that you just need to run (from theano's root) `python doc/scripts/docgen.py --nopdf --cache`. This will create an `html` directory with the documentation files, that you can then check (open `index.html` and browse the documentation as usual) before committing.